images/common/utils.sh

```
@@ -243,6 +243,12 @@ function openvpn_config_download {
 		${API_INTERNAL}/controller/vpn/checksum/$UUID/?key=$KEY
 	tar xzf vpn.tar.gz
 	chmod 600 *.pem
+
+	# Dynamically locate the config file and standardize its name for supervisord
+	CONF_FILE=$(ls *.conf 2>/dev/null | head -n 1)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)